### PR TITLE
Improve instructions for choosing actions

### PR DESCRIPTION
**Pull request by busywhistling**
Also apply code style (VS Code) to the bash script & correct the statement in the GPT prompt

### CODE REVIEW

**Comment by thomas-endres-tng** [comment]
Could you separate the formatting from the actual changes? It is hard to see the real changes in all these formatting diffs.

**Comment by busywhistling** [comment]
Sorry @thomas-endres-tng, my VS Code setup auto formats code on save, I should probably be more wary of this. Anyway, a nice little exercise in git to rebase my work upon the merged pull request.

File: please.sh [hunk 1]
```diff
@@ -300,16 +260,25 @@ display_menu() {
...
     copy_to_clipboard
-  else
     exit 0
-  fi
+    ;;
+  "A" | "a")
+    exit 0
+    ;;
+  *)
+    echo "Invalid choice. Please choose I, C or A."
```
Comment (thomas-endres-tng) [inline please.sh:277]: Hmm, don't know if this is a good idea.  I would at least remove the "Invalid choice part and mention the arrow keys".

File: README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ please list all files smaller than 1MB in the current folder, \
 💡 Command:
   find . -maxdepth 1 -type f -size -1M -exec wc -l {} + | sort -n -k1'
 
-❗ What should I do? [use arrow keys to navigate]
-> [I] Invoke   [C] Copy to clipboard   [A] Abort
+❗ What should I do? [use initials to select]
```
Comment (thomas-endres-tng) [inline README.md:22]: We should mention arrow keys AND initials, probably, since both will work.

**Review by thomas-endres-tng — CHANGES_REQUESTED** [review]

File: please.sh (hunk 1 above)
Comment (busywhistling) [reply please.sh:276]: Ahh, it was already implemented! I thought from the instructions that only the arrow selection worked, probably this PR is then redundant...

File: README.md (hunk 2 above)
Comment (thomas-endres-tng) [reply README.md:22]: I think it is not redundant, we should just list all options ;-)

**Pull request merged**